platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/main/java/com/swirlds/platform/test/consensus/framework/Util.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.test.consensus.framework;
+
+import com.swirlds.common.utility.Threshold;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class Util {
```

Comment:
A more descriptive name than "util" is appreciated.  searching can't get to this file without looking at paths.

Reply:
renamed